package.json

```diff
@@ -16,65 +16,62 @@
     "node": ">=4.0.0"
   },
   "scripts": {
-    "lint:bin": "eslint ./bin",
-    "lint:lib": "eslint ./lib",
-    "lint:test": "eslint ./__mocks__",
+    "lint": "eslint \"**/*.js\"",
+    "lint:codeOnly": "eslint \"{lib,bin,__mocks__}/**/!(__testfixtures__)/*.js\" \"{lib,bin,__mocks__}/**.js\"",
     "lint:staged": "lint-staged",
-    "lint": "npm run lint:bin && npm run lint:lib && npm run lint:test",
-    "install-commit-validator": "fit-commit-js install",
-    "test": "jest --coverage --runInBand"
+    "pretest": "npm run lint",
+    "test": "jest --coverage"
   },
+  "lint-staged": {
+    "{lib,bin,__mocks__}/**/!(__testfixtures__)/**.js": [
+      "eslint --fix",
+      "git add"
+    ]
+  },
+  "pre-commit": "lint:staged",
   "dependencies": {
     "babel-code-frame": "^6.22.0",
-    "babel-core": "^6.21.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-stage-3": "^6.17.0",
-    "chalk": "^1.1.3",
-    "cross-spawn": "^5.0.1",
-    "diff": "^3.2.0",
-    "enhanced-resolve": "^3.0.2",
-    "fit-commit-js": "^0.3.1",
-    "global-modules": "^0.2.3",
-    "got": "^6.6.3",
-    "inquirer": "^2.0.0",
-    "interpret": "^1.0.1",
-    "jscodeshift": "^0.3.30",
-    "listr": "^0.11.0",
-    "loader-utils": "^0.2.16",
+    "babel-core": "^6.25.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-stage-3": "^6.24.1",
+    "chalk": "^2.0.1",
+    "cross-spawn": "^5.1.0",
+    "diff": "^3.3.0",
+    "enhanced-resolve": "^3.3.0",
+    "global-modules": "^1.0.0",
+    "got": "^7.1.0",
+    "inquirer": "^3.2.0",
+    "interpret": "^1.0.3",
+    "jscodeshift": "^0.3.32",
+    "listr": "^0.12.0",
+    "loader-utils": "^1.1.0",
     "lodash": "^4.17.4",
     "p-each-series": "^1.0.0",
     "p-lazy": "^1.0.0",
-    "prettier": "^1.2.2",
+    "prettier": "^1.5.3",
     "recast": "git://github.com/kalcifer/recast.git#bug/allowbreak",
-    "resolve-cwd": "^2.0.0",
-    "supports-color": "^3.1.2",
-    "webpack": "^2.5.1",
-    "webpack-addons": "^1.1.2",
-    "yargs": "^6.5.0",
-    "yeoman-environment": "^1.6.6",
+    "rx": "^4.1.0",
+    "supports-color": "^4.2.0",
+    "webpack": "^3.3.0",
+    "webpack-addons": "1.1.2",
+    "webpack-addons-ylvis": "0.0.34",
+    "yargs": "^8.0.2",
+    "yeoman-environment": "^2.0.0",
     "yeoman-generator": "git://github.com/ev1stensberg/generator.git#Feature-getArgument"
   },
   "devDependencies": {
-    "ajv": "^4.11.3",
-    "ajv-keywords": "^1.5.1",
-    "babel-cli": "^6.18.0",
-    "babel-eslint": "^6.1.2",
-    "babel-jest": "^19.0.0",
-    "babel-polyfill": "^6.20.0",
-    "eslint": "^3.12.2",
-    "eslint-plugin-node": "^3.0.5",
-    "jest": "^19.0.2",
-    "lint-staged": "^3.2.8",
+    "ajv": "^5.2.2",
+    "ajv-keywords": "^2.1.0",
+    "babel-cli": "^6.24.1",
+    "babel-eslint": "^7.2.3",
+    "babel-jest": "^20.0.3",
+    "babel-polyfill": "^6.23.0",
+    "eslint": "^4.2.0",
+    "eslint-plugin-node": "^5.1.0",
+    "eslint-plugin-prettier": "^2.1.2",
```

Comment:
I see this dependency but I could not find the reference to it anywhere. Did you forget to add it to `eslintrc`?